packages/ui-react-utils/src/omitProps.ts

```diff
@@ -26,34 +26,35 @@
  * ---
  * category: utilities/react
  * ---
- * Return an object with the remaining props after propTypes (and additionally specified props) omitted.
+ * Return an object with the remaining props after the given props are omitted.
  *
- * Automatically excludes the following props: 'theme', 'children', 'className', 'style', 'styles', 'makeStyles', 'themeOverride'
+ * Automatically excludes the following props:
+ * 'theme', 'children', 'className', 'style', 'styles', 'makeStyles', 'themeOverride'
  * @module omitProps
- * @param {Object} props React component props
- * @param {Object|Array<string>} propTypes React component propTypes or the list of allowed prop keys
- * @param {Array} exclude an optional array of prop names to exclude
- * @returns {Object} props object without the excluded props
+ * @param props The object to process
+ * @param propsToOmit list disallowed prop keys or an object whose
+ *        keys will be omitted.
+ * @param exclude an optional array of disallowed prop names to omit
+ * @returns props object without the excluded props
  * @module omitProps
  */
 function omitProps<T extends Record<string, any>>(
   props: T,
-  propTypesOrAllowedPropList: Record<string, any> | string[],
+  propsToOmit: Record<string, any> | string[],
```

Comment:
this made it so hard to understand, this param specifies what to omit, not what is allowed